agent/engine/engine_unix_integ_test.go

```diff
@@ -189,11 +191,9 @@ func TestPortForward(t *testing.T) {
 	// Kill the existing container now to make the test run more quickly.
 	containerMap, _ := taskEngine.(*DockerTaskEngine).state.ContainerMapByArn(testTask.Arn)
 	cid := containerMap[testTask.Containers[0].Name].DockerID
-	client, _ := docker.NewClient(endpoint)
-	err = client.KillContainer(docker.KillContainerOptions{ID: cid})
-	if err != nil {
-		t.Error("Could not kill container", err)
-	}
+	client, _ := sdk.NewClientWithOpts(sdk.WithHost(endpoint))
+	err = client.ContainerKill(context.Background(), cid, "SIGKILL")
```

Comment:
It is preferable to use context.TODO here and tests below so that it can be canceled.

Reply:
Oh, I had thought I had moved all of the context.Background(), let me fix those.